Testscripts/Linux/XDPDumpSetup.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+# This script installs XDP dump application
+
+# Helper Function
+# Install dependencies for XDP
+function Install_XDP_Dependencies(){
+
+    if [ -z "${1}" ]; then
+        LogErr "ERROR: must provide install ip to Install_XDP_Dependencies()"
+        SetTestStateAborted
+        exit 1
+    fi
+
+    local install_ip="${1}"
+
+    CheckIP ${install_ip}
+    if [ $? -eq 1 ]; then
+        LogErr "ERROR: must pass valid ip to Install_XDP_Dependencies()"
+        SetTestStateAborted
+        exit 1
+    fi
+
+    DISTRO=$(ssh ${install_ip} lsb_release -is)
+    VERSION=$(ssh ${install_ip} lsb_release -sr)
+    DIST_VERSION="${DISTRO}_${VERSION}"
+    LLVM_VERSION="-6.0"
+    # Get the repository name from the distro version
```

Comment:
after source utils.sh. we have $DISTRO_NAME and $DISTRO_VERSION, some distro does't have command by default lsb_release.